pom.xml

```diff
@@ -820,12 +820,12 @@
         <version.org.wso2.orbit.javax.xml.bind>2.3.1.wso2v1</version.org.wso2.orbit.javax.xml.bind>
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
-        <identity.governance.version>1.11.11</identity.governance.version>
+        <identity.governance.version>1.11.18</identity.governance.version>
```

Comment:
Bump governance version once https://github.com/wso2-extensions/identity-governance/pull/887 is merged.

Reply:
Bumped with https://github.com/wso2/identity-api-server/pull/755/commits/6d4675c3dfa2519ead38020dd79865ab1138f2b3